README.md

```diff
@@ -28,7 +28,7 @@ Create a new WebSocket connection using the following.
 - Password is optional.  
 
 ```js
-const obs = new OBSWebSocket('address', 'password');
+const obs = new OBSWebSocket({ address: 'localhost:4444', password: '$up3rSecretP@ssw0rd' });
```

Comment:
Oh, duh, of course. The fact that I've been up and working since 6AM is showing.